website/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,504 @@
+- [Manage the operator using the Operator Lifecycle Manager](#manage-the-operator-using-the-operator-lifecycle-manager)
+  - [Generate an operator manifest](#generate-an-operator-manifest)
+  - [Testing locally](#testing-locally)
+  - [Promoting operator standards](#promoting-operator-standards)
+- [Conclusion](#conclusion)
+
+## Overview
+
+The [Operator Framework][org_operator_framework] ([intro blog post][site_blog_post]) is an open source toolkit to manage Kubernetes native applications, called operators, in an effective, automated, and scalable way. Operators take advantage of Kubernetes's extensibility to deliver the automation advantages of cloud services like provisioning, scaling, and backup/restore while being able to run anywhere that Kubernetes can run.
+
+This guide shows how to build a simple [memcached][site_memcached] operator and how to manage its lifecycle from install to update to a new version. For that, we will use two center pieces of the framework:
+
+* **Operator SDK**: Allows your developers to build an operator based on your expertise without requiring knowledge of Kubernetes API complexities.
+* **Operator Lifecycle Manager**: Helps you to install, update, and generally manage the lifecycle of all of the operators (and their associated services) running across your clusters.
+
+## Build an operator using the Operator SDK
+
+**BEFORE YOU BEGIN:** links to the Operator SDK repo in this document are pinned to the `master` branch. Make sure you update the link such that it points to the correct Operator SDK repo version, which should match this repo's version or the `operator-sdk version` being used. For example, if you are using `operator-sdk` v0.12.0, update all links from this repo to the SDK repo with `master -> v0.12.0`. Otherwise you may see incorrect information.
+
+The Operator SDK makes it easier to build Kubernetes native applications, a process that can require deep, application-specific operational knowledge. The SDK not only lowers that barrier, but it also helps reduce the amount of boilerplate code needed for many common management capabilities, such as metering or monitoring.
+
+This section walks through an example of building a simple memcached operator using tools and libraries provided by the Operator SDK. This walkthrough is not exhaustive; for an in-depth explanation of these steps, see the SDK's [user guide][doc_sdk_user_guide].
+
+**Requirements**: Please make sure that the Operator SDK is [installed][doc_sdk_install_instr] on the development machine. Additionally, the Operator Lifecycle Manager must be [installed][doc_olm_install_instr] in the cluster (1.8 or above to support the apps/v1beta2 API group) before running this guide.
+
+### Create a new project
+
+1. Use the CLI to create a new `memcached-operator` project:
+
+
+```sh
+$ mkdir -p $GOPATH/src/github.com/example-inc/
+$ cd $GOPATH/src/github.com/example-inc/
+$ export GO111MODULE=on
+$ operator-sdk new memcached-operator
```

Comment:
This doc conflicts with the existing Go quickstart guide, which uses the new kubebuilder-style CLI. We need to make sure this is marked as deprecated.

Reply:
Just saw this got closed, disregard.